en/mulesoft/modules/ROOT/pages/ms_composer_sage_intacct_reference.adoc

```diff
@@ -0,0 +1,225 @@
+
+== Triggers
+
+include::_partials/trigger-definition.adoc[]
+
+New Customer::
+
+Checks for a new Customer record.
+
+New or Updated Customer::
+
+Checks for new or updated Customer record.
+
+New Order::
+
+Checks for a new Order record.
+
+New or Updated Order::
+
+Checks for a new or updated Order record.
+
+== Actions
+
+include::_partials/action-definition.adoc[]
+
+Create Attachment::
+
+Creates a new Attachment record. In Sage Intacct, attachments are entities that contain files. An attachment object can be empty or have files. You can add files to an attachment when you create the record or you can add attachments later. Provide a value for the following:
+* *Attachment Name*
+* *Folder*
+* When adding files to an attachment, use a list as the source of the files, and specify a value in the following fields:
+** *File Binary Data*
+** *File Extension*
+** *File Name*
+
+Update Attachment::
+
+Updates an Attachment record. When updating an attachment, you can update the attachment properties or add new files from a list. Provide a value for *Attachment ID*.
+
+Get Attachments::
+
+Returns a list of Attachment records. To filter the list, provide a value for *Attachment ID*, *Attachment Name*, or *Folder Name*. To return the entire list, leave the fields blank.
+
+Create Attachment Folder::
+
+Creates a new folder for attachments. Provide a value for *Name*.
+
+Get Attachment Folders::
+
+Returns a list of attachment folders. To filter the list, provide a value for *Folder Name* to filter. To return the entire list, leave the fields blank.
+
+Create Customer::
+
+Create a new Customer record. Provide a value for the following:
+
+* *Customer ID*
+* *Primary Contact*
+* *Print As*
+
+Update Customer::
+
+Update a Customer record. Provide a value for *ID*.
+
+Get Customer::
+
+Returns a list of Customer records. To filter the list, provide a value for *Customer ID* or *Customer Name*. To return the entire list, leave the fields blank.
+
+Delete Customer::
+
+Deletes a Customer record. Provide a value for *Customer ID*.
+
+Create Order::
+
+Creates a new Order record and returns a value for the *Key* field. Provide a value for the following:
+
+* *Order Type*
+* *Date Created*
+* *Transaction Currency*
+* *Base Currency*
+* *Exchange Rate Type*
+* In the *Entries* > *Sotransitem* section, provide a value in the following:
+  ** *Sotransitem List*
+  ** *Item ID*
+  ** *Quantity*
+  ** *Location ID* +
+Note that the *Location ID* field is determined by the numbers preceding the location description. For example, if the location ID is `100-United States` then enter `100` in the *Location ID* field.
+
+  ** *Price*
+  ** *Unit*
+  ** If your order contains inventory type items or a mix of inventory and non-inventory items, provide a value for *Warehouse ID*.
```

Comment:
For consistency with the other items in the list, I would flip this and start with *Warehouse ID*, if your order...